_includes/header.html

```diff
@@ -1,12 +1,8 @@
 <div class="ui teal inverted menu">
-  {% if include.page == 'home' %}
-  <a href="/" class="active item">
-    {% else %}
-    <a href="/" class="item">
-      {% endif %}
-      <i class="home icon"></i>
-      Home
-    </a>
+  <a href="/" class="{% if include.page == 'home' %}active {% endif %}item">
+    <i class="home icon"></i>
+    Home
+  </a>
 
     {% if include.page == 'home' %}
```

Comment:
If I'm not mistaken 'home' is the old main page so there's not really any reason for keeping this section.

Reply:
not sure what part you are saying to remove. Wanna give some line numbers? havent looked at this in a while and github showing things a bit weird for me.

Reply:
L7 to L26.